### PR TITLE
fix #77284 buzz.ie

### LINKED ISSUE #77284: www.buzz.ie
### Issue URL (Incorrect Blocking)
 [https://www.buzz.ie/movies-tv/comedies-binge-watch-netflix-255263](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.buzz.ie%2Fmovies-tv%2Fcomedies-binge-watch-netflix-255263)
### Comment
Username: @benbristow
> Cannot scroll because of cookie modal

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | EdgeChromium
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> Filter unblocking search ads and self-promotion<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>Thai Ads Filters

### PR DESCRIPTION
**Pull request by wout-codes**
#77284

### CODE REVIEW

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -723,7 +723,7 @@ alltravnik.ru,seahair.ru##.flat-cm-cookie
 wix.com##div[data-hook="ccsu-banner-wrapper"]
 modacruz.com##div[class="_wi _Y"]
 strato.de#$#.consent { display: none !important; }
-strato.de#$#.noscroll { overflow: visible !important; }
+buzz.ie,strato.de#$#.noscroll { overflow: visible !important; }
```
[the remark this answers is not in the document]
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:726]: What's the difference? My rules work too.

[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: `#$#` rules should be referred to their domains. Another thing is that these rules are more accurate.

[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Also when you solve the issue, you should take a look at obsolete rules.

[the remark this answers is not in the document]
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Okay.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -723,7 +724,6 @@ alltravnik.ru,seahair.ru##.flat-cm-cookie
 wix.com##div[data-hook="ccsu-banner-wrapper"]
 modacruz.com##div[class="_wi _Y"]
 strato.de#$#.consent { display: none !important; }
-strato.de#$#.noscroll { overflow: visible !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:726]: Don't remove it. Please revert the change.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: I also like them more, but sometimes scriptlets can't be executed properly in the extension. And remember to use an easier way to solve the problem. CSS < scriptlets.

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @zloyden Btw, is the do the devs even update the extension? It's just like that for a few months as seen.

[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Ofc, they do, but now they are working on some global changes.

[the remark this answers is not in the document]
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: What do you mean by global changes?

[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: You'll see it :). But it's better to ask @maximtop if you have specific questions.

[the remark this answers is not in the document]
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Ok.
